CHANGELOG.md

```diff
@@ -2,455 +2,534 @@
 
 ## [Unreleased]
 
+Yuri Izmer:
```

Comment:
The reformatting of the file is somehow unexpected

Reply:
Oops

Reply:
It was formatted automatically by vscode after "search and replace"